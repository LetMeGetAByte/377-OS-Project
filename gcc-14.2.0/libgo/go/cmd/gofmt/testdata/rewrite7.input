version https://git-lfs.github.com/spec/v1
oid sha256:6b3753dbe4bab05c14e12a44550776177cf9acafbbcfd473a22ef333e118ba4f
size 472
