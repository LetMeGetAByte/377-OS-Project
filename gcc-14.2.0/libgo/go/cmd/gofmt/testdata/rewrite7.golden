version https://git-lfs.github.com/spec/v1
oid sha256:d6fc83971481803b35bd0369caec4dd3ed5ea3b934220e1dd7431d149e287b30
size 466
